pyspecProcScripts/__init__.py

```diff
@@ -1,12 +1,18 @@
-from .phasing import zeroth_order_ph,hermitian_function_test,ph1_real_Abs,determine_sign
-from .first_level.fake_data import fake_data
+from .phasing import (
+    zeroth_order_ph,
+    hermitian_function_test,
+    ph1_real_Abs,
+    determine_sign,
+)
```

Comment:
unsure why this wasn't here before - can always make change in a separate PR but this seems simple enough since I'm already editing this file